apps/frontend/src/api/discover.ts

```diff
@@ -0,0 +1,8 @@
+import useFetch from 'use-http';
+import { DiscoverResponse } from '@asap-hub/model';
+
+import { API_BASE_URL } from '../config';
+import { useFetchOptions } from './util';
+
+export const useDiscover = () =>
+  useFetch<DiscoverResponse>(`${API_BASE_URL}/discover`, useFetchOptions(), []);
```

Comment:
Please do not `useFetch` directly anymore, instead `useGetList` or (I assume in this case) `useGetOne`. See structure of one of the other components that `useGetOne` for how to consume.
Side note: I love using the `use` that is part of a hook's name as the verb for a sentence ^^